frontend/src/components/TeamRegistration.js

```diff
@@ -349,6 +396,69 @@ const TeamRegistration = () => {
                   </div>
                 )}
               </For>
+
+              <For each={tournamentQuery.data?.partial_teams}>
+                {team => (
+                  <div class="mb-4 flex items-center justify-between gap-x-4 border-b pb-4">
+                    <div class="flex items-center gap-x-4">
+                      <img
+                        src={team.image ?? team.image_url}
+                        class="h-8 w-8 rounded-full"
+                      />
+                      <span class="font-medium">
+                        {team.name}{" "}
+                        <span class="me-2 rounded bg-red-100 px-2.5 py-0.5 text-xs font-medium text-red-800 dark:bg-red-900 dark:text-red-300">
+                          Partial
+                        </span>
```

Comment:
Should we should this only to the team admin ? I don't think all users need to know the teams' payment status right ?

Reply:
Hmm I thought it be better cause OPs also would wanna know and it avoids misconception

Reply:
right, makes sense. We can think about an ops role for the Hub later on maybe, for things like this, and if there's a need. Fine for now